lib/matplotlib/units.py

```diff
@@ -51,17 +51,29 @@ def default_units(x, axis):
 
 
 class AxisInfo(object):
-    """information to support default axis labeling and tick labeling, and
-       default limits"""
+    """
+    Information to support default axis labeling, tick labeling, and
+    default limits.
+    """
     def __init__(self, majloc=None, minloc=None,
                  majfmt=None, minfmt=None, label=None,
                  default_limits=None):
         """
-        majloc and minloc: TickLocators for the major and minor ticks
-        majfmt and minfmt: TickFormatters for the major and minor ticks
-        label: the default axis label
-        default_limits: the default min, max of the axis if no data is present
-        If any of the above are None, the axis will simply use the default
+        Parameters
+        ----------
+        majloc, minloc :
```

Comment:
„The colon must be omitted if the type is absent.“ https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections
Also in the following lines.